docs/security/content_trust.md

```diff
@@ -0,0 +1,156 @@
+<!--[metadata]>
++++
+alias = [TBD]
+title = "Content trust in Docker"
+description = "Enabling content trust in Docker"
+keywords = ["trust, security, docker,  documentation"]
+[menu.main]
+parent= "smn_administrate"
++++
+<![end-metadata]-->
+
+## Enabling content trust in Docker
+
+Docker 1.8 enables content trust. Content trust means that all operations with a remote registry enforce  client-side signing and verification of content tags. Currently content trust is opt-in. We are planning to change this behavior to opt-out in a future release. For now, in order to have trusted operations with Docker you will need to either `export DOCKER_CONTENT_TRUST=1` , or use the `--disable-content-trust=false` flag.
+
+When enabled, content trust will ensure that all operations using a remote registry enforce the signing and verification of tags. In particular `push`, `pull`, `build`, `create` and `run` will only operate on tags that either have content signatures or explicit content hashes.
+
+#### Pulling trusted content
+Let's try to pull a remote repository on the Docker hub that does not have content-trust data:
+
+```
+➜ export DOCKER_CONTENT_TRUST=1
+➜ docker pull diogomonica/alpine
+Using default tag: latest
+no trust data available
+```
+
+As we can see, since we have docker content trust enabled, we are not allowed to pull from a repository that does not have signed content.
+
+#### Pushing trusted content
+
+To create signed content for a specific tag you simply have to push to a repository, providing a content tag. However, if this is the first time you are pushing to a remote repository with content trust enabled, two things will happen:
+
+ - A new root key will be generated and you will be asked to provide a phassphrase for this key
+ - A new tagging pair will be generated, and you will also be asked to provide a passphrase for these keys
```

Comment:
is one key genereated for each `docker login` user I have? or will it use the same key all the time?